utility/src/test/java/org/oppia/android/util/networking/NetworkConnectionUtilProdImplTest.kt

```diff
@@ -148,6 +149,45 @@ class NetworkConnectionUtilProdImplTest {
     assertThat(networkConnectionUtil.getCurrentConnectionStatus()).isEqualTo(NONE)
   }
 
+  @Test
+  fun testLocalConnectionName_activeWifiConnection_returnsWifi() {
+    networkConnectionTestUtil.setNetworkInfo(
+      status = ConnectivityManager.TYPE_WIFI,
+      networkState = NetworkInfo.State.CONNECTED
+    )
+    assertThat(context.resources.getString(LOCAL.connectionName)).isEqualTo(
+      context.resources.getString(
+        R.string.network_connection_local
+      )
+    )
+  }
+
+  @Test
+  fun testLocalConnectionName_activeCELLULARConnection_returnsCELLULAR() {
```

Comment:
Please fix the ending of this test name.

Also, I expect the beginning of this & the other new test names will need to be updated after renaming connectionName.

Reply:
Fixed